bin/tflocal

```diff
@@ -220,6 +221,7 @@ def generate_s3_backend_config() -> str:
         "key": "terraform.tfstate",
         "dynamodb_table": "tf-test-state",
         "region": get_region(),
+        "use_path_style": "true",
```

Comment:
I'm not sure everyone would be happy if we pin this option here or force them to add it explicitly with false when they don't need it.
The best option probably if we introduce a similar logic like this instead:
https://github.com/localstack/terraform-local/blob/07643c4364c97c1089d97a2a11dd329553d1ca7f/bin/tflocal#L136

_(Sidenote: users use tflocal in combination with real s3 backends too, so we must not break those setups either.)_

Reply:
Agreed, I will work on that!

Reply:
I set the default to false and I am reusing the `use_s3_path_style` method so it should be easier for a user to use different combinations of backend/s3 target